src/martialarts.cpp

```diff
@@ -23,9 +23,17 @@ void load_technique(JsonObject &jo)
         tec.name = _(tec.name.c_str());
     }
 
-    JsonArray jsarr = jo.get_array("messages");
-    while (jsarr.has_more()) {
-        tec.messages.push_back(_(jsarr.next_string().c_str()));
+    if( jo.has_member( "messages" ) ) {
+        JsonArray jsarr = jo.get_array("messages");
+        tec.player_message = jsarr.get_string( 0 );
+        tec.npc_message = jsarr.get_string( 1 );
+        if( !tec.player_message.empty() ) {
+            tec.player_message = _(tec.player_message.c_str());
+        }
+        tec.npc_message = jsarr.get_string( 1 );
```

Comment:
Likely copy/paste failure: looks like you're `get`ting`_string( 1 )` twice accidentally.